.changeset/rotten-wasps-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@theguild/tailwind-config': minor
```

Comment:
I used `minor` here as we're on 0.x version.
This is breaking if we ever used .dark class to override the color scheme per section.